### PR TITLE
New App: Simple National Pokedex

### PR DESCRIPTION
**Pull request by laurenkopac**
Displays a Pokemon at random from the National Pokedex by default. Ability to choose region (Gen I - VII).

### CODE REVIEW

**Comment by laurenkopac** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by laurenkopac** [comment]
recheck

File: apps/natdex/natdex.star [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        MAX = "649"
+    elif dex_region == "Kalos":
+        MIN = "650"
+        MAX = "721"
+    elif dex_region == "Alola":
+        MIN = "722"
+        MAX = "809"
+    else:
+        pass
+    resp = http.get("https://www.random.org/integers/?num=1&min=" + MIN + "&max=" + MAX + "&col=1&base=10&format=plain&rnd=new1")
```
Comment (betterengineering) [inline apps/natdex/natdex.star:77]: We added the `random` module after you created this PR 🤦  but it could eliminate this HTTP request

**Review by betterengineering — APPROVED** [review]

**Comment by betterengineering** [comment]
oh, this was never added to `apps/apps.go`! I'm going to merge this PR and fix it up in `main`.

**Pull request merged**